base/abstractdict.jl

```diff
@@ -136,6 +136,29 @@ values(a::AbstractDict) = ValueIterator(a)
 Return an iterator over `key => value` pairs for any
 collection that maps a set of keys to a set of values.
 This includes arrays, where the keys are the array indices.
+
+# Examples
+```jldoctest
+julia> a = Dict(zip(["a", "b", "c"], [1, 2, 3]))
+Dict{String, Int64} with 3 entries:
+ "c" => 3
+ "b" => 2
+ "a" => 1
+
+julia> pairs(a)
```

Comment:
I wonder how useful this specific example is when it literally just returns the input unmodified. How often would you call `pairs` on a `Dict` in practice?

```jl
ulia> (;a=1, b=2) |> pairs |> collect
2-element Vector{Pair{Symbol, Int64}}:
 :a => 1
 :b => 2

julia> (;a=1, b=2) |> collect
2-element Vector{Int64}:
 1
 2
```

might be something that is useful to show?

Reply:
Sure, why not?

Reply:
I do think people might naively expect `pairs(d)` to return a `Vector{Pair}` even if `d` is a `Dict`, which is why I included it.

Reply:
Normally you would iterate this thing, right? So perhaps one of the examples might show that, like
```
julia> foreach(println, pairs(["a", "b", "c"]))
1 => "a"
2 => "b"
3 => "c"
```